cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -213,11 +213,11 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
         }
 
         val packages = mutableListOf<Package>().apply {
-            if (PackageType.PROJECTS in packageTypes) {
+            if (PackageType.PROJECT in packageTypes) {
```

Comment:
Nit: I think this commit should then also adjust the `Downloading ${packageTypes.joinToString(" and ")} from ORT` code from the previous commit by adding a plural "s" after `}`.